tests/modeladmin/models.py

```diff
@@ -28,6 +28,39 @@ class Concert(models.Model):
     ), blank=True)
 
 
+@python_2_unicode_compatible
```

Comment:
My thinking was that it's fine to combine the tests, e.g. 
```
class EmployeeProfileAdmin(ModelAdmin):
    list_filter = ['employee__employeeinfo__description', '...']
```
should also work. Is there any problem with that?